### PR TITLE
Enable request caching migration

### PR DESCRIPTION
**Pull request by Stanvirk**

### CODE REVIEW

File: guide/migration/20-to-30.md [hunk 1]
```diff
@@ -98,6 +98,20 @@ services.AddOpenIddict()
     });
 ```
 
+## Enable request caching
```
Comment (kevinchalet) [inline guide/migration/20-to-30.md:101]: Should the section title be updated to something that doesn't imply it's a mandatory feature? Maybe "If applicable, use the new request caching activation methods"?

**Comment by kevinchalet** [comment]
Thanks for your PR!

File: guide/migration/20-to-30.md (hunk 1 above)
Comment (Stanvirk) [reply guide/migration/20-to-30.md:101]: Good idea. Changed description. Is it better now?

**Comment by kevinchalet** [comment]
Merged, thanks for your contribution! :tada:

**Pull request merged**